providers/flagd/pkg/service/grpc/tests/mocks/mock-service-client.go

```diff
@@ -0,0 +1,138 @@
+package mocks
```

Comment:
Why not use https://github.com/golang/mock for this?

Reply:
updated tests to use https://github.com/golang/mock